src/tests/ftest/launch.py

```diff
@@ -1376,14 +1377,16 @@ def install_debuginfos():
     # installation
     path = os.path.sep + os.path.join('usr', 'share', 'spdk', 'include')
     if os.path.islink(path):
-        cmds.append(["sudo", "rm", "-f", path])
+        cmd_list.append(["sudo", "rm", "-f", path])
 
     if USE_DEBUGINFO_INSTALL:
         yum_args = [
             "--exclude", "ompi-debuginfo", "libpmemobj", "python", "openmpi3"]
-        cmds.append(["sudo", "yum", "-y", "install"] + yum_args)
-        cmds.append(["sudo", "debuginfo-install", "--enablerepo=*-debuginfo",
-                     "-y"] + yum_args + ["daos-server", "gcc"])
+        cmd_list.append(timeout + ["sudo", "dnf", "-y", "install"] + yum_args)
+        cmd_list.append(
+            timeout +
+            ["sudo", "debuginfo-install", "--enablerepo=*-debuginfo", "-y"] +
```

Comment:
This is going to intermittently mean failing to get a stacktrace, due to `yum` timing out and getting killed.  It would be better to just use `dnf`:

```suggestion
            ["sudo", "dnf", "debuginfo-install", "-y"] +
```

Reply:
Fixed